test/unit/CowOperator/setup.js

```diff
@@ -46,12 +46,14 @@ async function setup() {
   await daiAggregator.setLatestAnswer(0.0002 * 1e18); // 1 dai = 0.0002 eth, 1 eth = 5000 dai
   const stethAggregator = await ChainlinkAggregatorMock.deploy();
   await stethAggregator.setLatestAnswer(parseEther('1')); // 1 steth = 1 eth
+  const usdcAggregator = await ChainlinkAggregatorMock.deploy();
+  await usdcAggregator.setLatestAnswer(0.0002 * 1e18); // 1 usdc = 0.0002 eth, 1 eth = 5000 dai
```

Comment:
`usdc` was deployed with 6 decimals (see line 32).

Reply:
because this is USDC/ETH and not ETH/USDC the number of decimals here is 18 (value is in ETH)

Check the chainlink prod oracle as an example https://data.chain.link/ethereum/mainnet/stablecoins/usdc-eth#operator-alpha_chain